auth_mellon_cache.c

```diff
@@ -1,7 +1,7 @@
 /*
  *
  *   auth_mellon_cache.c: an authentication apache module
- *   Copyright � 2003-2007 UNINETT (http://www.uninett.no/)
+ *   Copyright � 2003-2007 UNINETT (http://www.uninett.no/)
```

Comment:
Character encoding issue, can you please fix ?

Reply:
fixed